ui/admin/app/components/form/role/manage-scopes/index.hbs

```diff
@@ -0,0 +1,88 @@
+<Rose::Form
+  class='full-width role-manage-scopes-form'
+  @onSubmit={{fn this.submit @submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  as |form|
+>
+
+  <Hds::Form::Toggle::Field
+    @value={{this.KEYWORDS.THIS}}
+    name={{this.KEYWORDS.THIS}}
+    checked={{includes this.KEYWORDS.THIS this.selectedItems}}
+    disabled={{form.disabled}}
+    {{on 'change' this.toggleField}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.role.scope.form.this.label'}}</F.Label>
+    <F.HelperText>{{t 'resources.role.scope.form.this.help'}}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  <Hds::Form::Toggle::Field
+    @value={{this.KEYWORDS.CHILDREN}}
+    name={{this.KEYWORDS.CHILDREN}}
+    checked={{includes this.KEYWORDS.CHILDREN this.selectedItems}}
+    disabled={{or
+      form.disabled
+      (includes this.KEYWORDS.DESCENDANTS this.selectedItems)
+    }}
+    {{on 'change' this.toggleField}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.role.scope.form.children.label'}}</F.Label>
+    <F.HelperText>{{if
+        @model.scope.isGlobal
+        (t 'resources.role.scope.form.children.help.0')
+        (t 'resources.role.scope.form.children.help.1')
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  {{#if @model.scope.isGlobal}}
+    <Hds::Form::Toggle::Field
+      @value={{this.KEYWORDS.DESCENDANTS}}
+      name={{this.KEYWORDS.DESCENDANTS}}
+      checked={{includes this.KEYWORDS.DESCENDANTS this.selectedItems}}
+      disabled={{or
+        form.disabled
+        (includes this.KEYWORDS.CHILDREN this.selectedItems)
+      }}
+      {{on 'change' this.toggleField}}
+      as |F|
+    >
+      <F.Label>{{t 'resources.role.scope.form.descendants.label'}}</F.Label>
+      <F.HelperText>{{t
+          'resources.role.scope.form.descendants.help'
+        }}</F.HelperText>
+    </Hds::Form::Toggle::Field>
+  {{/if}}
+
+  {{! TODO: Add correct href value inline link}}
+  {{#if this.showAlert}}
+    <Hds::Alert @type='compact' as |A|>
+      <A.Description>
+        {{t 'resources.role.scope.messages.keywords-selected.description'}}
+        <Hds::Link::Inline @color='secondary' @href={{(doc-url 'role')}}>
+          {{t 'resources.role.scope.messages.keywords-selected.link'}}
+        </Hds::Link::Inline>
+      </A.Description>
+    </Hds::Alert>
+  {{/if}}
+
+  <Hds::Button
+    @text={{t 'resources.role.scope.actions.add-custom-scopes.text'}}
+    @color='secondary'
+    @route='scopes.scope.roles.role'
```

Comment:
This is a temp route until we have the route for add-custom-scopes?

Reply:
yup correct! I forgot to leave a todo comment 😅